src/components/common/CartSVG.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+function CartSVG() {
+  return (
+    <svg
```

Comment:
I think there is a way to declare svg file and use them in react, you can look at that but don't chage this